api/views/course.py

```diff
@@ -184,20 +184,33 @@ def course_grade_book(self, request, pk):
         for event in events.all():
             teams.extend(event.team_set.all())
 
+        from course.models.models import UserQuestionJunction, Submission
```

Comment:
The imports should be at the top. It that causes circular import we probably have done something wrong.